blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionHandler.java

```diff
@@ -99,19 +99,32 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
 
             requestCount.inc();
             final String tenantId = request.getHeader("tenantId");
-            JSONMetricsContainer jsonMetricsContainer = null;
+            JSONMetricsContainer jsonMetricsContainer;
+            List<Metric> metrics;
+
             final Timer.Context jsonTimerContext = jsonTimer.time();
 
             final String body = request.getContent().toString(Constants.DEFAULT_CHARSET);
             try {
                 jsonMetricsContainer = createContainer(body, tenantId);
 
                 if (jsonMetricsContainer == null) {
-                    log.warn(ctx.getChannel().getRemoteAddress() + " No valid metrics");
+                    log.warn(ctx.getChannel().getRemoteAddress() + " No valid json for container.");
                     DefaultHandler.sendResponse(ctx, request, "No valid metrics", HttpResponseStatus.BAD_REQUEST);
                     return;
+                } else if (jsonMetricsContainer.getJsonMetrics().isEmpty()) {
+                    log.warn(ctx.getChannel().getRemoteAddress() + " No valid json metrics to parse.");
+                    DefaultHandler.sendResponse(ctx, request, "Error converting JSON payload to metric objects", HttpResponseStatus.BAD_REQUEST);
+                    return;
```

Comment:
Just to clarify - I thought if they didn't send any metrics we were going to send a 200.  Was that for a different scenario?